docs/product-getting-started-guide/src/main/asciidoc/test-speed-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='testing_speed_limit-proc']
+= Testing the speed limit scenario
+
+The following chapter instructs you how to test the speed limit scenario using the data that you specified when you created the speed violation guided decision table.
```

Comment:
See my comment in the Adding the Violation data object data fields section, about intro statements.

Reply:
done throughout